haddock-api/resources/html/Linuwial.std-theme/linuwial.css

```diff
@@ -390,6 +390,8 @@ pre {
   margin: 1em 0 0 0;
   background-color: #f7f7f7;
   overflow: auto;
+  border: 1px solid #ddd;
```

Comment:
this looks nice.

Reply:
Ah, the #444 is the same as header's bottom border. Than this makes sense. :+1:

Reply:
Yeah, I was trying to match the header's bottom border. Initially I had it as `gainsboro` (to match the vertical line on the left), but that was a bit too light.

Reply:
I'm quite happy with current state. It's clearly an improvement for me.

Reply:
Agree, looks nice 👍